apps/builder/app/shared/sync/sync-stores.ts

```diff
@@ -192,25 +195,89 @@ const syncStoresState = (name: SyncEventSource, publish: Publish) => {
   };
 };
 
+const handshakeAndSyncStores = (
```

Comment:
handshake(source, publish, whenBothReady)

Reply:
sync is called immediately, we just postpone action send